### PR TITLE
Add Snippet for maxPods and podsPerCore

### PR DESCRIPTION
**Pull request by mburke5678**
Inspired by https://issues.redhat.com/browse/OCPBUGS-20332

In researching the above Jira, I noticed that there were two places that repeated much of the same information with a slight skew. Created a snippet of the info to prevent skew. Did not change the text.

### CODE REVIEW

**Review by xenolinux — COMMENTED** [review]
One suggestion for consideration otherwise LGTM!

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: snippets/nodes-pods-core-max-pods.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+[source,yaml]
+----
+kubeletConfig:
+  podsPerCore: 10
+----
+
+Setting `podsPerCore` to `0` disables this limit. The default is `0`.
+`podsPerCore` cannot exceed `maxPods`.
+
+`maxPods` sets the number of pods the node can run to a fixed value, regardless
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply snippets/nodes-pods-core-max-pods.adoc:47]: @xenolinux Good catches all. I actually didn't review the text, assuming that the text had passed peer review previously. Perhaps I should have reviewed it!

**Pull request merged**